### DISCUSSION TITLE
Caption width or height exceeds limit

### DISCUSSION THREAD

**Discussion by KennyBrunton** [ImageMagick/ImageMagick#2705] [Help] [answered]
Hello,

I'm having an issue with something that used to work fine.  Hosting company updated to ImageMagick 6.9.10-23 recently I believe and there seems to be an issue with captions.

The command below fails - and yet removing a few characters and it works and creates a file 880x273 which is well within the width and height limits of 16KP

Any thoughts would be greatly appreciated.

`convert -background lightblue -fill blue -pointsize 23 -interline-spacing 15 -size 880x caption:'This is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, This is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, this is a test, abc, abc, abc, abc, abc, abc, Remove this->sdfjdsklflkjlkjlkjlkjlk! ' test.png`

**Comment by fmw42** [comment, marked as answer]
There has been numerous changes to caption: and label: So it is possible you have a version that might not be working as good as possible.  I get the following with IM 6.9.11.34. 

![test](https://user-images.githubusercontent.com/13458218/95661684-00605080-0ae6-11eb-81d2-4b5a9d8b1571.png)

Perhaps you should upgrade

**Reply by KennyBrunton** [reply]
Thanks Fred, looks like the software is the issue. I've left it in the hands of my hosting provider - hopefully something can be done about it.
